app/src/main/java/com/example/nigeriatelemedicineapp/api/services/GetIdentifierService.kt

```diff
@@ -0,0 +1,12 @@
+package com.example.nigeriatelemedicineapp.api.services
+
+import com.example.nigeriatelemedicineapp.api.models.Identifier
+import retrofit2.Call
+import retrofit2.http.GET
+import retrofit2.http.Url
+
+interface GetIdentifierService {
+
+  @GET
+  fun getIdentifier(@Url url: String ="https://openmrs-staging.callnigeriandoc.com/openmrs/module/idgen/generateIdentifier.form?source=1&username=admin&password=Admin123"): Call<Identifier>
```

Comment:
This hard-coded URL is defined both here and in BaseURL.kt. Is there any way we can have a single source of truth for the openmrs URL, preferably through some kind of configuration variable?